RocketChat/Rocket.Chat.ReactNative
# [FIX] Avatar's size on display mode change

## Proposed changes

Updates Avatar container component to be able to update when the size changes, removes unused prop `avatarSize` as well
## Issue(s)	


## How to test or reproduce

1. Go to `Settings > Display`
2. Change between `Condensed` and `Expanded`
3. Go to `Chats`
4. Scroll through the room's list

Avatar's size should resize properly
## Screenshots

## Types of changes




## Checklist



## Further comments
